internal/pkg/testutils/utils.go

```diff
@@ -269,8 +269,6 @@ func CreateContainer(netconf, podName, podNamespace, ip string) (containerID str
 }
 
 // Create container with the giving containerId when containerId is not empty
-//
-// Deprecated: Please call CreateContainerNamespace and then RunCNIPluginWithID directly.
```

Comment:
Why did you remove this comment? I think we don't want people using the function in the test code.

Reply:
One of the linters enforces `Deprecated` comments, but I spoke to @ozdanborne and he told me to ignore it and stick with the current call.